common/buildcraft/builders/tile/TileBuilder.java

```diff
@@ -246,6 +247,7 @@ public void writePayload(int id, PacketBufferBC buffer, Side side) {
         }
     }
 
+    @SideOnly(Side.CLIENT)
```

Comment:
The argument "side" is guaranteed to never be CLIENT on the server. So its safe.